README.md

```diff
@@ -29,7 +29,12 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+# # def offer_rose (person)
+#   @person = person
+#   puts "would you like a rose, #{@person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
+# end
+#
+# offer_rose ('young prince')
```

Comment:
:+1: This works, but you don't need an instance variable (i.e., a variable with `@`). You can just reference the `person` argument directly in the `puts` statement.